### PR TITLE
Use symlinks option

### PR DESCRIPTION
**Pull request by florentdupont**
Pull Request for issue #147.
Notice that this option doesn't affect the behavior of the `symlinks` parameter and _shared_ folder.

### CODE REVIEW

File: libraries/chef_artifact.rb [hunk 1]
```diff
@@ -284,6 +292,22 @@ def encrypted_data_bag_item(data_bag, data_bag_item)
...
       end
+
+      # Create and Write content of the ".symlinks" file. This file contains a reference
+      # to the reference of release version which "current" is pointing to.
+      #
+      # @param deploy_to_dir [String] root directory containing the ".symlinks" file.
+      # @param current_version_pointing_to [String] current release version referenced by "current"
+      def write_current_symlink_to(deploy_to_dir, current_version_pointing_to)
+        symlinks_file = ::File.join(deploy_to_dir, ".symlinks")
+        if ::File.exist? symlinks_file
```
Comment (KAllan357) [inline libraries/chef_artifact.rb]: Please add parenthesis to this method call.

File: providers/deploy.rb [hunk 2]
```diff
@@ -241,7 +279,7 @@ def copy_artifact
   recipe_eval do
     execute "copy artifact" do
       command Chef::Artifact.copy_command_for(cached_tar_path, release_path)
-      user new_resource.owner
+      user  new_resource.owner
```
Comment (KAllan357) [inline providers/deploy.rb]: Looks like an extra space found its way in here.

File: providers/deploy.rb [hunk 3]
```diff
@@ -147,20 +147,58 @@ def load_current_resource
   end
 
   recipe_eval do
-    if Chef::Artifact.windows?
-      # Needed until CHEF-3960 is fixed.
-      symlink_changing = current_symlink_changing?
-      execute "delete the symlink at #{new_resource.current_path}" do
-        command "rmdir #{new_resource.current_path}"
-        only_if {Chef::Artifact.symlink?(new_resource.current_path) && symlink_changing}
+    if !@new_resource.use_symlinks
```
Comment (KAllan357) [inline providers/deploy.rb:150]: Is it possible to simplify this if block with the on starting on line 171 and just use if/else? I'm thinking that might make this section a bit more clear.

**Comment by florentdupont** [comment]
I made some cleaning on the lines you commented.

**Comment by KAllan357** [comment]
Thank you! :+1:

**Pull request merged**